openmetadata-ui/src/main/resources/ui/src/components/MyData/CustomizableComponents/CustomiseLandingPageHeader/CustomiseLandingPageHeader.tsx

```diff
@@ -236,7 +237,9 @@ const CustomiseLandingPageHeader = ({
                     role="button"
                     tabIndex={0}
                     onClick={() => {
-                      navigate(`/${data.entityType}/${data.name}`);
+                      navigate(
```

Comment:
can you confirm if this works when fullyQualifiedName contains special characters.
I believe we need to encode and pass, but double check.